decidim-core/lib/decidim/content_parsers/user_group_parser.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+module Decidim
+  module ContentParsers
+    # A parser that searches user groups mentions in content.
+    #
+    # A word starting with `@` will be considered as a possible mention if
+    # they only contains letters, numbers or underscores. If the `@` is
+    # followed with an underscore, then it is not considered.
```

Comment:
Can you document the why are we ignoring mentions starting with `@_`?

Reply:
I have no idea, to be honest! That was like this before my changes.

Reply:
Here, I would like some help. I don't have any clue why we don't accept underscores in nicknames. It seems deliberate but it is easy to change, we just need to change the regexp from 
```ruby
MENTION_REGEX = /\B@([a-zA-Z0-9]\w*)\b/.freeze
```
to
```ruby
MENTION_REGEX = /\B@(\w*)\b/.freeze
```

Maybe @decidim/product can say way nicknames starting with "_" (underscore) are ignored?